gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -66,6 +67,7 @@
         LiveDataReactiveStreams.fromPublisher(dataRepository.getActiveProjectOnceAndStream());
     this.navigator = navigator;
     this.addFeatureClicks = PublishSubject.create();
+    this.downloadTileClicks = PublishSubject.create();
```

Comment:
When I first read this I thought it was referring to "toggle offline tile clicks", but it appears it's the click to show the offline imagery UI. It might be more obvious if we called it something like "showOfflineImagerySelectorClicks()". 

Ideally I think we may want to actually have the selector be a separate View and ViewModel so we don't need to worry about interactions with other features on the map UI (click feature, location lock, etc). Any thoughts on this?

Reply:
I think keeping them separate would definitely be a better design. I can give this a shot.